lib/dl_api_lib/dl_api_lib/connector_registrator.py

```diff
@@ -59,6 +60,7 @@ def register_connector(cls, connector: Type[ApiConnector]) -> None:
                 factory_cls=mqm_setting_item.factory_cls,
             )
         register_is_forkable_source(backend_type=backend_type, is_forkable=connector.is_forkable)
```

Comment:
@altvod, take a look please
This is unused at the moment, do we need this?

Reply:
If it's not used, then let's remove it

Reply:
done